node-zerox/src/utils.ts

```diff
@@ -144,11 +145,38 @@ export const convertFileToPdf = async ({
   const outputPath = path.join(tempDir, outputFilename);
 
   try {
-    const pdfBuffer = await convertAsync(inputBuffer, ".pdf", undefined);
+    const pdfBuffer = await convertAsync(inputBuffer, "pdf", undefined);
```

Comment:
I think we might want to keep the `.pdf`. Seems like the `.` is required per the docs:
<img width="1130" alt="Screen Shot 2024-09-03 at 4 32 47 PM" src="https://github.com/user-attachments/assets/7d42f8f0-6ff3-4250-a5ab-d54426989c73">